src/interfaces/vault.ts

```diff
@@ -177,8 +177,25 @@ export class VaultInterface<T extends ChainId> extends ServiceInterface<T> {
         return vaultContract.deposit(amount, overrides);
       }
     } else {
-      // ZAPPER!
-      throw new SdkError("deposit:zapper not implemented.");
+      const zapInParams = await this.yearn.services.zapper.zapIn(
+        account,
+        token,
+        amount,
+        vault,
+        overrides.gasPrice?.toString() || "0",
+        0.01
+      );
+
+      const transaction: TransactionRequest = {
+        to: zapInParams.to,
+        from: zapInParams.from,
+        gasLimit: overrides.gasLimit?.toString() || "0",
+        gasPrice: zapInParams.gasPrice,
+        data: zapInParams.data as string,
+        value: zapInParams.value as string
+      };
+
+      return this.ctx.provider.write.getSigner(account).sendTransaction(transaction);
```

Comment:
`sendTransaction` fills in the missing properties of `TransactionRequest` e.g. nonce, however nonce is specified in the overrides, should I use that? I suppose if I use nonce from the overrides and it's null it'll then be filled in by `sendTransaction`

Reply:
`signer` is already declared on line 169